src/Symfony/Bundle/WebProfilerBundle/composer.json

```diff
@@ -4,7 +4,7 @@
     "description": "Symfony WebProfilerBundle",
     "keywords": [],
     "homepage": "https://symfony.com",
-    "license": "MIT",
+    "license": "(MIT and CC-BY-3.0)",
```

Comment:
Composer allows to define multiple licenses using an array. See https://getcomposer.org/doc/04-schema.md#license